recipes/swig/all/conandata.yml

```diff
@@ -1,11 +1,19 @@
 sources:
+  "4.1.0":
+    url: "https://github.com/swig/swig/archive/refs/tags/v4.1.0.tar.gz"
+    sha256: "5b9313b1af5edfcea158a389520be266f013bc9be4ce933d79a30c5659ba99fe"
   "4.0.2":
     url: "https://github.com/swig/swig/archive/rel-4.0.2.tar.gz"
     sha256: "81d7ce78371f378a3299ddc5aea1da9a6178f325dcabb695d1b742f9e24a0fa6"
   "4.0.1":
     url: "https://github.com/swig/swig/archive/rel-4.0.1.tar.gz"
     sha256: "2eaf6fb89d071d1be280bf995c63360b3729860c0da64948123b5d7e4cfb6cb7"
 patches:
+  "4.1.0":
+    - base_path: "source_subfolder"
```

Comment:
we have new https://github.com/conan-io/conan-center-index/blob/master/docs/conandata_yml_format.md#patches-fields

If you one and close the linter was merged jsut recently to check them 😉